src/views/studio/l10n.json

```diff
@@ -29,8 +29,7 @@
 
     "studio.projectsHeader": "Projects",
     "studio.addProjectsHeader": "Add Projects",
-    "studio.addProject": "Add",
-    "studio.addProjectPlaceholder": "Project URL",
+    "studio.addProject": "Add by URL",
```

Comment:
I'm not sure if we need to create a new id when updating a string that has already been merged in- investigating...

Reply:
turns out this is the correct way to do it (keep the same id and update the string).

Reply:
thanks for investigating!